packages/xo-web/src/xo-app/index.js

```diff
@@ -114,6 +116,17 @@ const BODY_STYLE = {
     signedUp: !!state.user,
   }
 })
+@provideState({
+  initialState: () => ({ checkXoaCall: 0}),
+  effects: {
+    addOneCheckXoaCall(){
+      this.state += 1
+    }
+  },
+  computed: {
+    checkXoa: (state,props) => checkXoa
```

Comment:
You need to explicitly access the count to ensure subscription to `checkXoaCall`, and as a consequence, recomputation in case of change.